content/problems/bio1/index.tsx

```diff
@@ -893,4 +894,14 @@ export const bio1Problems: BIO1ProblemInfo[] = [
     solutionAuthors: ["Adwaya Gupta"],
     component: <Trail />,
   },
+  {
+    difficulty: "Insane",
+    display: "String Rewriting",
+    original: "https://olympiad.org.uk/papers/2007/bio/bio07exam.pdf",
+    year: 2007,
+    tags: ["Q3", "Dynamic Programming", "Implementation", "Graphs"],
```

Comment:
Q3 doesn't go as a tag anymore. `question` is an attribute on the object itself.

Reply:
I fixed it. Why is it still jambloating 😭 AAAAA